lib/msf/core/exploit/remote/tcp_server.rb

```diff
@@ -26,7 +26,6 @@ def initialize(info = {})
 
     register_advanced_options(
       [
-        OptString.new('ListenerComm', [ false, 'The specific communication channel to use for this service']),
```

Comment:
This was unnecessary since the same option is inherited by the included SocketServer mixin.